docs/sample.tslint.json

```diff
@@ -1,168 +0,0 @@
-{
```

Comment:
This file is essentially redundant documentation at this point and an unnecessary burden to maintain, so I deleted it.